birdhouse/components/weaver/config/canarie-api/canarie_api_monitoring.py.template

```diff
@@ -26,7 +26,7 @@ SERVICES['Weaver'] = {
     "monitoring": {
         "Weaver": {
             'request': {
-                'url': 'http://weaver:4001/'
+                'url': 'http://weaver:4001/${WEAVER_MANAGER_NAME}/'
```

Comment:
Pour votre info, Ouranos production https://pavics.ouranos.ca/canarie/node/service/stats have the old                 `'url': 'http://weaver:4001/` and it reports success.

Reply:
Ouranos server is on 1.42.2, so it doesn't include the changes from https://github.com/bird-house/birdhouse-deploy/releases/tag/2.8.0 and https://github.com/bird-house/birdhouse-deploy/releases/tag/2.7.3 that, when combined cause the problem, but not detected when tested separately on their own.

https://github.com/bird-house/birdhouse-deploy/blob/18929112c0ef14a210d47cb5a31f3242d2deee8c/birdhouse/components/weaver/docker-compose-extra.yml#L21-L25

Reply:
@mishaschwartz ⬆️ 

This why you didn't notice it as wrong. 
The healthcheck was created before that change (so it worked), while I was testing Weaver before it was integrated, so never saw the error.

Reply:
Ah right! That makes sense